hack/builder/Dockerfile

```diff
@@ -92,7 +92,7 @@ RUN set -x && \
     rm -rf /test-infra && \
     rm -rf /go && mkdir /go
 
-RUN pip3 install --upgrade j2cli operator-courier==1.3.0
+RUN pip3 install --upgrade j2cli operator-courier==2.1.11
```

Comment:
This is actually addressed in https://github.com/kubevirt/kubevirt/pull/5087, I will rebase once that gets merged but it's basically duplicated otherwise the tests would not pass (not completely true because the updated image is already in quay).

Anyway, we need to bump the operator-courier because v1.3.0 it does not support api-extensions v1 but just v1beta1.